src/index.ts

```diff
@@ -137,14 +144,18 @@ class Egnyte {
      * @param password the passworf for auth
```

Comment:
Small changes like this are a good chance to refactor and fix typos and things. Like this out-of-date JsDoc annotation that describes parameters that no longer exist.